src/index.ts

```diff
@@ -1308,6 +1335,11 @@ class Remarkable implements RemarkableApi {
   async delete(hash: string, refresh: boolean = false): Promise<HashEntry> {
     return await this.move(hash, "trash", refresh);
   }
+  
+  /** permanently delete an entry */
+  async purge(hash: string, refresh: boolean = false): Promise<void> {
```

Comment:
I would make this have a similar return type, e.g. `[SimpleEntry, string]` although I left a comment on the bulk purge return type as well.

Reply:
yeah, lets wait until the more important details are _hashed_ out.